### ISSUE TITLE
Manually increment play_count

### ISSUE THREAD

**Issue by Naahuel** [cmus/cmus#489]
I couldn't really find a more appropriate place to post this. It's not an issue or even a feature request, just something I'd like to know if it's possible.

I've used Foobar2000 for many years (it's still my favorite player, by far, but cmus is awesome and I'm starting to love it a lot).

With Foobar2000 I used a plugin called "[Playback Statistics](http://wiki.hydrogenaud.io/index.php?title=Foobar2000:Components/Playback_Statistics_v3.x_(foo_playcount))", which provided play counts and many other cool stats.  

Basically, I have years of play counts and I'm able to export it from Foobar2000 (in XML with encoded ID's, but I'll figure that out later), but no way of importing to cmus.

Could I somehow achieve this? 

Thanks!

**Comment by flyingmutant** [comment]
Right now this is not possible, unfortunately — cmus stores play counts in its binary cache file, and there are no import/export tools for that.

**Comment by Naahuel** [comment]
Yes, thank you, I know that. I'm glad you added the feature request label, I started thinking after I posted this that this could actually be something that would interest other people.

I'm guessing I'll write something to achieve this via display_program for now. Trying to manually edit the cache file sounds like a lot of work.
